caller-workflow-templates/cdeployment-ecs-caller.yml

```diff
@@ -0,0 +1,36 @@
+name: CD pipeline for ECS
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push tag event
+  push:
+    tags:
+      - v*
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  VERSION: 'latest'
+
+jobs:
+  graasp-deploy-ecs-workflow:
+    name: CD caller template
+    uses: graasp/graasp-deploy/.github/workflows/cdeployment-ecs.yml@main
+    with:
+      ecs-task-definition: '.aws/<name>-test.json'
```

Comment:
Why does this finish in `-test`?

Reply:
This refers to the task definition template for the ECS task in the staging environment. Should it be changed to <name>-staging?

Reply:
This refers to the task definition template for the ECS task in the staging environment. I change it to <name>-staging.json